go.mod

```
@@ -1,3 +1,5 @@
 module todo-app3
```

```suggestion
module github.com/EgorBKV4/todo-app
```
должно быть такое же название как и путь в гитхаб репозитории